data_structures/binary_trees/words_alphabetical.c

```diff
@@ -0,0 +1,241 @@
+/**
+ * @file
+ * @brief Printing the words contained in a file "file.txt" in alphabetical
```

Comment:
Provide a Wikipedia link in Markdown format (if available/possible).
If there's no Wikipedia link, add another web source for algorithm explanation. 🙂

Reply:
ok, i am on it